### PR TITLE
travis: actually test the pull requests

### PR DESCRIPTION
**Pull request by AlexKornitzer**
Just testing a theory that spawned from the issues with getting the MDMP PR to test correctly.

### CODE REVIEW

**Comment by AlexKornitzer** [comment]
Theory seems to be correct.

File: .travis.yml [hunk 1]
```diff
@@ -29,57 +29,57 @@ addons:
       - zlib1g-dev
       - swig
 env:
-  - TESTS="export R2PM_GITDIR=${TRAVIS_BUILD_DIR} && r2pm -i armthumb"
-  - TESTS="export R2PM_GITDIR=${TRAVIS_BUILD_DIR} && r2pm -i baleful"
+  - TESTS="export R2PM_GITDIR='/home/travis/build/radare' && r2pm -i armthumb"
```
Comment (radare) [inline .travis.yml]: thats ugly

Comment (AlexKornitzer) [inline .travis.yml]: Yeah I know but ${TRAVIS_BUILD_DIR} gives the incorrect path. Are you able to set a new env variable in the travis build system?

**Comment by radare** [comment]
you can do that in .travis.yml

**Comment by AlexKornitzer** [comment]
Ah was treating it like pure yml, let me clean it up.

**Comment by AlexKornitzer** [comment]
Right finally got it! Sorry about blitzing the build server. Additional observations that have arisen, probably due to a lack of r2r tests:

- `r2api-python` is broken but is reporting as passed (not sure how long this has been broken for)
- `retdec` is broken but is reporting as passed (not sure how long this has been broken for)
- `yara-r2` is broken and reporting as failed

**Comment by Maijin** [comment]
This is totally wrong, if you remove the export you are testing master not the PRs

**Comment by Maijin** [comment]
Ah ok nvm you are doing it afterwards, cleaner then my bad

**Comment by AlexKornitzer** [comment]
Yeah, the issue with it originally was that it was using the master as the TRAVIS_BUILD_DIR puts you into the radare2-extras repo rather than a dir above it, thus it always cloned from master from what I can tell.

**Comment by Maijin** [comment]
My guessed is that TravisCI changed a little since last time that's why it's broken now

**Comment by AlexKornitzer** [comment]
Forgot to remove the matrix header I added when experimenting, should be good to go now.

**Pull request merged**